migrations/Version20150924204612.php

```diff
@@ -14,7 +14,7 @@ class Version20150924204612 extends AbstractMigration
     public function up(Schema $schema)
     {
         $todo = $schema->getTable(Table::TODO);
-        $todo->addColumn('deadline', 'datetime', ['default' => null, 'notnull' => false]);
+        $todo->addColumn('deadline', 'string', ['default' => null, 'notnull' => false, 'length' => 30]);
     }
 
```

Comment:
Good question!
Basically because of [this limitation](http://doctrine-dbal.readthedocs.org/en/latest/reference/known-vendor-issues.html#datetimetz)

The purpose of read tables is to provide information in a query-friendly way. ISO8601 is a common standard and I would like to store datetime without the need to map it again.